src/public/client.ts

```diff
@@ -95,6 +95,13 @@ class MappingItem {
           ${data.gitLink}
         </small>
       </div>
+      <a
+        href="/manage/${data.fullDomain}"
+        class="btn btn-sm btn-outline-primary mr-3 manageButton"
```

Comment:
Just a suggestion, instead of camelCase for CSS classes, maybe use a BEM style? `manageButton => manage-button`